features/page_objects/back_office/new/back_office_app.rb

```diff
@@ -154,6 +154,10 @@ def waste_carrier_sign_in_page
     @last_page = WasteCarrierSignInPage.new
   end
 
+  def user_invite_page
```

Comment:
I'm just being picky, but can you put this in alphabetical order in the file?

Reply:
Sure XD